app/models/invoice.rb

```diff
@@ -10,4 +10,5 @@ class Invoice < ApplicationRecord
   def self.incomplete_invoices
     Invoice.where(status: "in progress").order(created_at: :asc)
```

Comment:
I don't think you need to use `Invoice` here since it is a class method for Invoices - it will call `.where`... on self, which is invoices. By leaving it out it gives you the ability to call the method on a group of invoices (like the invoices belonging to a particular merchant) instead of all invoices.